packages/ndla-editor/src/structure/ItemNameBar.js

```diff
@@ -97,8 +115,22 @@ const ItemNameBar = ({
   id,
   level,
   isVisible,
+  favoriteSubjectIds,
+  toggleFavorite,
 }) => (
   <StyledItemBar level={level} highlight={highlight}>
+    {favoriteSubjectIds && (
+      <RoundIcon
+        onClick={() => toggleFavorite()}
+        smallIcon={
+          favoriteSubjectIds.includes(id) ? (
+            <Star color="#fcba03" />
```

Comment:
Ville vurdert å gjøre denne om til en konstant som sier hva farge det er :-)

Reply:
Hva tenkte du på?
Som en enkel `const StarColorOrange` elns eller i `color.ts`?

Reply:
@gunnarvelle @N4G1 hva er vanlig praksis i NDLA på dette? Kan vi legge til slike farger i `color.ts` eller er det bare "NDLA brand" farger som skal i den filen?

Reply:
Godt spørsmål, jeg vet faktisk ikke hva policy'en er på å legge ting til `colors.ts` og lignende

Reply:
Føles jo ut som om det naturligvis burde ligge med de andre fargene, slik at man kan bruke de samme flere steder om det er behov for det.

Reply:
Gunnar kom innom og sa det var ok å legge den til i colors.ts, så da legger jeg den inn der

Reply:
sweet, da lærte vi alle noe nytt i dag @N4G1 @MaPoKen 😅